automation/taskcluster/lib/tasks.py

```diff
@@ -582,7 +581,7 @@ def _craft_artifacts_from_variant(variant):
 
 def _craft_apk_full_path_from_variant(variant):
     architecture, build_type = get_architecture_and_build_type_from_variant(variant)
-    postfix = '-unsigned' if build_type.startswith('release') else ''
+    postfix = '' if build_type == 'Debug' else '-unsigned'
```

Comment:
Do you confirm Raptor has now `variant in ('armRaptor', 'aarch64Raptor)` which means:
 * `architecture in ('arm', 'aarch64')`
 * `build_type == 'Raptor'`

Therefore, a raptor build ends with `-unsigned.apk`, right?

Reply:
Note: good point on checking the `build_type`, it is lowercased from `get_arch_and...`, so I should be checking "Debug". I'll run a new build and double-check the results

Otherwise, the new `$archRaptor` have builds that end in `-unsigned`, exactly :)
```
$ ./gradlew printBuildVarients
variants: ["aarch64Raptor","aarch64Debug","aarch64Nightly","aarch64Beta","x86Raptor","x86Debug","x86Nightly","x86Beta","armRaptor","armDebug","armNightly","armBeta"]
```